mgmtfn/k8splugin/README.md

```diff
@@ -8,35 +8,18 @@ Networking and Policy can be used for Pod inter-connectivity in a Kubernetes clu
 This step-by-step procedure will guide you through a minimal experience of creating
 a Kubernetes cluster with Contiv networking and applying policy between pods.
 
-#### Step 1: Clone and build kubernetes
-
-```
-$ mkdir ~/go/src/github.com/k8s
-$ cd ~/go/src/github.com/k8s
-$ git clone https://github.com/kubernetes/kubernetes
-$ cd kubernetes; make
-```
-
-Note: If the build fails, please retry.
-
-#### Step 2: Clone contrib repo
+#### Step 1: Clone contrib and netplugin repos
 
 ```
+$ mkdir -p ~/go/src/github.com/k8s
 $ cd ~/go/src/github.com/k8s
 $ git clone https://github.com/jojimt/contrib -b contiv
-```
-
-#### Step 3: Clone and build netplugin
 
-```
 $ cd ~/go/src/github.com/k8s
 $ git clone https://github.com/contiv/netplugin
-
-$ cd netplugin
-$ make run-build
 ```
 
-#### Step 4: Set proxies if your network requires http/https proxy.
+#### Step 2: Set proxies if your network requires http/https proxy.
```

Comment:
IMHO, this step can be just a note instead of a step. Not everyone will have a http proxy